VisualStudioCode/README.md

```diff
@@ -18,3 +18,50 @@ After that, you can launch VS Code from your terminal:
 
 * `code .` will open VS Code in the current directory
 * `code myfile.txt` will open `myfile.txt` in VS Code
+
+### Python Extension
```

Comment:
A description to each extension would be nice, so the reader knows why they should install that specific extension.

Reply:
Sure